rpc/doc.md

```diff
@@ -288,21 +288,30 @@ curl -d '{"id": 2, "jsonrpc": "2.0", "method":"get_tip_block_number","params": [
 }
 ```
 
-# local_node_id
+# local_node_info
 
-Returns the local node id.
+Returns the local node information.
 
 ## Examples
 
 ```shell
-curl -d '{"id": 2, "jsonrpc": "2.0", "method":"local_node_id","params": []}' -H 'content-type:application/json' 'http://localhost:8114'
+curl -d '{"id": 2, "jsonrpc": "2.0", "method":"local_node_info","params": []}' -H 'content-type:application/json' 'http://localhost:8114'
 ```
 
 ```json
 {
     "jsonrpc": "2.0",
-    "result": "/ip4/0.0.0.0/tcp/8115/p2p/QmdSxB6iTcbhj6gbZNthvJrwRkJrwnsohNpVixY4FtcZwv",
-    "id": 2
+    "result": {
+        "addresses": [
+            {
+                "address": "/ip4/0.0.0.0/tcp/12344/p2p/QmWRU2NSro4wKgVbFX6y8SPFkcJ1tE2X5xzk9msMhdRmdS",
```

Comment:
Can `/p2p/xxxx` be omitted here? because it's duplicated with `node_id`.

Reply:
After considering, I think we can leave it here because it is easy to copy-paste or parse in script, a typical node may have 2 addresses(local address and public address), it could have more addresses if it has multiple network interfaces, even it has multi public addresses the duplicate p2p phase is not big deal.